tests/OperationProcessor.spec.ts

```diff
@@ -224,4 +224,27 @@ describe('OperationProcessor', async () => {
     const didDocument = await operationProcessor.resolve(didUniqueSuffix);
     expect(didDocument).toBeUndefined();
   });
+
+  it('should return undefined for deleted did', async () => {
+    const numberOfUpdates = 1;
```

Comment:
Sanity: did you intend to just have a 1 update?

Reply:
No, debugging artifact. Fixed.